### PR TITLE
Update SourceLink article to refer to NuGet for symbol file advice

### LINKED ISSUE #9966: Update SourceLink guidance for NuGet symbol server
The [main NuGet guidance page](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/nuget#symbol-packages) was updated for the new NuGet symbol servers, but the [page about SourceLink](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink) wasn't. Although the recommendation seems to be still the same (include pdb files in main NuGet package), the reason is now a different one. Therefore this page should be updated too.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5db5ae44-5ee9-ed62-7355-7c0cbc1bfc07
* Version Independent ID: f54c98e2-bbae-42c6-2103-e94d6ec0e931
* Content: [SourceLink and .NET libraries](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink)
* Content Source: [docs/standard/library-guidance/sourcelink.md](https://github.com/dotnet/docs/blob/master/docs/standard/library-guidance/sourcelink.md)
* Product: **dotnet**
* GitHub Login: @JamesNK
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by JamesNK**
Fixes #9966

### CODE REVIEW

File: docs/standard/library-guidance/sourcelink.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ You can use [NuGet Package Explorer](https://github.com/NuGetPackageExplorer/NuG
 > * <xref:System.Diagnostics.DebuggerStepThroughAttribute> instructs the debugger to step through the code instead of stepping into the code.
 > * <xref:System.Diagnostics.DebuggerBrowsableAttribute> controls whether a member is displayed in the debugger variable windows.
 
-**✔️ CONSIDER** including symbol files (`*.pdb`) in the NuGet package.
+**✔️ CONSIDER** publishing symbol files (`*.pdb`).
 
-> Ordinarily, you'd publish symbol files in a [symbol package](./nuget.md#symbol-packages). Currently the main public host for symbol packages doesn't support the portable symbol files (`*.pdb`) created by SDK-style projects, and symbol packages aren't useful.
+> Read more about [symbol files and symbol packages](./nuget.md#symbol-packages).
```
Comment (mairaw) [inline docs/standard/library-guidance/sourcelink.md]: Our boilerplate for this kind of link is something like For more information, see or For more information about x, see...

```suggestion
> For more information about symbol files and symbol packages, see [Symbol packages](./nuget.md#symbol-packages).
```

File: docs/standard/library-guidance/sourcelink.md [hunk 2]
```diff
@@ -29,9 +29,9 @@ You can use [NuGet Package Explorer](https://github.com/NuGetPackageExplorer/NuG
 > * <xref:System.Diagnostics.DebuggerStepThroughAttribute> instructs the debugger to step through the code instead of stepping into the code.
 > * <xref:System.Diagnostics.DebuggerBrowsableAttribute> controls whether a member is displayed in the debugger variable windows.
 
-**✔️ CONSIDER** including symbol files (`*.pdb`) in the NuGet package.
+**✔️ CONSIDER** publishing symbol files (`*.pdb`).
```
Comment (mairaw) [inline docs/standard/library-guidance/sourcelink.md]: would it be helpful to say "with your package"?

**Review by mairaw — APPROVED** [review]
LGTM. Left some comments for you to consider before merging.

File: docs/standard/library-guidance/sourcelink.md (hunk 2 above)
Comment (cremor) [reply docs/standard/library-guidance/sourcelink.md]: I like the generic wording that @JamesNK used here. That way this page doesn't have to be updated again should the recommendation on the "NuGet" page change in the future.

**Pull request merged**